app/views/declaration/summary/summary_page.scala.html

```diff
@@ -18,17 +18,24 @@
 @import controllers.declaration.routes._
 @import models.declaration.SupplementaryDeclarationData
 @import models.viewmodels.HtmlTableRow
+@import models.requests.JourneyRequest
 @import uk.gov.hmrc.play.views.html._
 @import views.html.declaration.summary._
 
-@(suppDecData: SupplementaryDeclarationData)(implicit appConfig: AppConfig, request: Request[_], messages: Messages)
+@(suppDecData: SupplementaryDeclarationData)(implicit appConfig: AppConfig, request: JourneyRequest[_], messages: Messages)
 
 @main_template(
     title = messages("supplementary.summary.title"),
     appConfig = appConfig) {
 
-
-    @components.back_link("/customs-declare-exports/declaration/warehouse")
+    @{if(request.choice.value == "STD")
```

Comment:
Please import object and use variable for `"STD"` when the type will change or something like this it will be easier to maintain